ospatch/zypper_patch.go

```diff
@@ -25,11 +28,14 @@ var (
 )
 
 type zypperPatchOpts struct {
-	categories   []string
-	severities   []string
-	withOptional bool
-	withUpdate   bool
-	runner       func(cmd *exec.Cmd) ([]byte, error)
+	categories       []string
+	severities       []string
+	withOptional     bool
+	withUpdate       bool
+	excludes         []string
+	exclusivePatches []string
+	runner           func(cmd *exec.Cmd) ([]byte, error)
```

Comment:
You can drop the runner now as we don't use it anymore

Reply:
done